utils/web_ui.py

```
@@ -3,7 +3,15 @@
 import threading
 import logging
 from queue import Queue
-from flask import Flask, render_template, jsonify, request, redirect, url_for, send_from_directory
+from flask import (
+    Flask,
+    render_template,
+    jsonify,
+    request,
+    redirect,
```

Ruff (0.11.9)

`flask.redirect` imported but unused

Remove unused import

(F401)